client/cypress/e2e/earn.cy.ts

```diff
@@ -211,13 +208,10 @@ Object.values(viewports).forEach(({ device, viewportWidth, viewportHeight, isDes
           );
           cy.get('[data-test=GlmLockNotification--success]').should('be.visible');
           cy.get('[data-test=GlmLockTabs__Button]').click();
-          cy.wait(5000);
+          // Waiting 2s is a way to prevent the effects of slowing down the e2e environment (data update).
```

Comment:
Waiting is required twice, here and in `moveEpoch`, LOC56? Can't waiting be encapsulated in `moveEpoch` util? E.g.:
```ts
...
  // Waiting 2s is a way to prevent the effects of slowing down the e2e environment (data update).
  cy.wait(2000);
  await cypressWindow.mutateAsyncMoveEpoch();
  // Waiting 2s is a way to prevent the effects of slowing down the e2e environment (data update).
  cy.wait(2000);
  // Manually taking a pending snapshot after the epoch shift ensures that the snapshot is taken. Passing epoch multiple times without manually triggering pending snapshot in a short period of time may cause the e2e environment to fail.
  await axios.post(`${env.serverEndpoint}snapshots/pending`);
  ...
  ```

Reply:
@aziolek done